Tools/ApplicationDebugger/array-transform/README.md

```diff
@@ -24,9 +24,9 @@ scheduler-locking and SIMD lane views.  The sample is intended
 for exercising the debugger, not for performance benchmarking.
 
 The debugger supports debugging kernels that run on the CPU, GPU, or
-accelerator devices.  For convenience, the `array-transform`
-code sample provides the ability to select the target device by passing the
-program `cpu`, `gpu`, or `accelerator` as the command-line argument.
+accelerator devices.  Use ONEAPI_DEVICE_SELECTOR environment variable
```

Comment:
Nit: Use *the* ONEAPI... environment variable

Reply:
fixed.